charts/comet-site-v1/Chart.yaml

```diff
@@ -16,4 +16,4 @@ maintainers:
   - name: Alexander Kaufmann
     email: alexander.kaufmann@vivid-planet.com
 
-version: 8.9.1
+version: 1.0.0
```

Comment:
All other charts have 0.0.0, this one has 1.0.0. Which one is correct?

Reply:
0.0.0 is correct, because changeset will set it to 1.0.0 when merged